### PR TITLE
add option to change packages on hold (packages/apt)

### PR DESCRIPTION
**Pull request by yfoelling**
##### SUMMARY
Added an option to be able to change held back packages (apt-mark hold).

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
`lib/ansible/modules/packaging/os/apt.py`

##### ADDITIONAL INFORMATION
Please check if the new option is everywhere it needs to be.

### CODE REVIEW

File: lib/ansible/modules/packaging/os/apt.py [hunk 1]
```diff
@@ -51,6 +51,11 @@
   default_release:
     description:
       - Corresponds to the C(-t) option for I(apt) and sets pin priorities
+  change_held_packages:
+    description:
+      - Corresponds to the C(--allow-change-held-packages) option for I(apt). It allows you to explicitly decide to upgrade packages you set on hold.
+    type: bool
+    version_added: "2.8"
```
Comment (bcoca) [inline lib/ansible/modules/packaging/os/apt.py:59]: missing 'default: false' in docs

**Review by bcoca — CHANGES_REQUESTED** [review]

**Comment by tinproject** [comment]
@yfoelling @bcoca Any new on this? I believe all the requested changes are already fulfilled.

(I'm pinging here because I was expecting this went into 2.8, but I could not found it on the changelog)

**Comment by samdoran** [comment]
!component =lib/ansible/modules/packaging/os/apt.py

**Comment by immesys** [comment]
This would be handy. Is there a reason it is stuck?

**Comment by samdoran** [comment]
This needs a changelog entry and integration tests (plus a rebase).

**Comment by Akasurde** [comment]
superseded by https://github.com/ansible/ansible/pull/73629

@yfoelling Thanks for the contribution.

**Pull request closed without merging**